### DISCUSSION TITLE
Why can't we switch from realisticStockPhoto_v10 to vjuggernautXL_version6Rundiffusion without restarting?

### DISCUSSION THREAD

**Discussion by stephantual** [lllyasviel/Fooocus#1600] [Q&A] [answered]
As you know you can use -run_realistic , which defaults to realisticStockPhoto_v10 , while the 'default' run command pre-loads vjuggernautXL_version6Rundiffusion . 

But we also know that to use 'other' models, we just need to place them in the model folder. And yet, even without modifying the config file, and having juggernautXL_version6Rundiffusion  AND realisticStockPhoto_v10 in that folder, i can't select them interchangeabley - i have to restart foocus. 

I was wondering why that was given the only official way to load models is by placing them in that folder? 

Thank you

**Comment by mashb1t** [comment, marked as answer]
I'm not sure if i got your point. You should be able to use them interchangeably, despite each of the presets loading different LoRAs, negative prompts, etc.. What is the exact issue here?

Make sure to click the "Refresh All Files" button (tab Advanced > Model) after adding a new model to be able to select it.
This is also where you select your model and change it in the select from Juggernaut to realisticStockPhoto or different.

Please let me know if you need further assistance and if yes, please provide maybe some screenshots/error messages/click-by-click instructions.

**Reply by stephantual** [reply]
I'm not in front of the computer now, but I'll try tomorrow. Thank you so much
